core/injector/application_injector.cpp

```diff
@@ -1043,6 +1043,8 @@ namespace {
             [](auto const &injector) { return get_grandpa_impl(injector); }),
         di::bind<consensus::grandpa::CatchUpObserver>.to(
             [](auto const &injector) { return get_grandpa_impl(injector); }),
+        di::bind<consensus::grandpa::NeighborObserver>.to(
```

Comment:
looks like not used

Reply:
You are right. But it can be usefull for future